plugin/pkg/admission/persistentvolume/label/admission.go

```diff
@@ -44,7 +49,8 @@ var _ = admission.Interface(&persistentVolumeLabel{})
 
 type persistentVolumeLabel struct {
 	*admission.Handler
-
+	client           internalclientset.Interface
```

Comment:
I would use a new informer instead of extending this one to help keep the functionality clear and distinct.